example-problems/no-such-method-error-signature-mismatch/pom.xml

```diff
@@ -39,6 +39,34 @@
           <mainClass>io.grpc.internal.App</mainClass>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
This change doesn't seem related to this PR.

Reply:
Sorry, missed git operation. Removed.